frontend/src/style.css

```diff
@@ -56,3 +56,19 @@ img {
   opacity: 0.2;
   pointer-events: none;
 }
+
+.inputStyle {
```

Comment:
De én ezt az egészt class-t kivenném és helyileg formáznám Bootstrap-es class-okkal, text-left (ami amúgy szerintem a default, hogy balra van rendezve), font-weight-normal stb.
De amúgy meg miért nem hasznájlátok a Bootstrap-es default radio formázást? https://getbootstrap.com/docs/4.6/components/forms/#checkboxes-and-radios